website/src/cwkw/cwkw-layout.css.ts

```diff
@@ -0,0 +1,18 @@
+import { style } from "@vanilla-extract/css"
+import * as baseLayout from "../layout.css"
+import { colors } from "./theme.css"
+
+export const header = style([baseLayout.header])
```

Comment:
If the style is just reusing another one, I think you can just do `export const header = baseLayout.header`. You could reduce this even further by doing `export { header, headerContents, siteTitle, siteLink } from "../layout.css"`